Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
                 sh '''
                     git clone "https://github.com/leil-io/sfstests"
                     cd sfstests
-                    git checkout v0.2.x
+                    git checkout v0.2.1
```

Comment:
Does it means that every time you change the version you need to create a PR in saunafs?

Reply:
Correct, the idea being if something breaks it can be reverted to a previous version, and also allowing me to test newer ones without breaking things.